app/src/components/inputs/inputUserSearch/inputUserSearch.jsx

```diff
@@ -61,51 +65,81 @@ const makeURL = (input, isAdmin, projectId) => {
   startUrl += `?${stringify(qsParams, { encode: false })}`;
   return startUrl;
 };
-const makeOptions = (options, projectId) => options.map(option => ({
-  userName: option.full_name || '',
-  userLogin: option.userId,
-  email: option.email || '',
-  disabled: Object.prototype.hasOwnProperty.call(option.assigned_projects, projectId),
-  isAssigned: Object.prototype.hasOwnProperty.call(option.assigned_projects, projectId),
-  userAvatar: getPhoto(option.userId),
-}));
+const makeOptions = (options, projectId) =>
+  options.map((option) => ({
+    userName: option.full_name || '',
+    userLogin: option.userId,
+    email: option.email || '',
+    disabled: Object.prototype.hasOwnProperty.call(option.assigned_projects, projectId),
+    isAssigned: Object.prototype.hasOwnProperty.call(option.assigned_projects, projectId),
```

Comment:
Is there any possibility that prototype of an `option.assigned_projects` has property `projectId`?